code/datums/elements/food/processable.dm

```diff
@@ -54,3 +62,22 @@
 		examine_list += span_notice("It can be turned into [amount_created] [initial(result_atom_type.name)]s with <b>[tool_behaviour_name(tool_behaviour)]</b>!")
 	else
 		examine_list += span_notice("It can be turned into \a [initial(result_atom_type.name)] with <b>[tool_behaviour_name(tool_behaviour)]</b>!")
+
+///Adds context sensitivy directly to the processable file for screentips
+/datum/element/processable/proc/on_requesting_context_from_item(
+	datum/source,
+	list/context,
+	obj/item/held_item,
+	mob/user,
+)
```

Comment:
```suggestion
/datum/element/processable/proc/on_requesting_context_from_item(datum/source, list/context, obj/item/held_item, mob/user)
```

you don't have THAT many arguments. also, might be worthwhile to DMdoc what each of these mean/do

Reply:
let me know if my documentation is off or wrong, but i think that theyre correct

Reply:
The format looks alright to my eyes